plugin-base/src/main/kotlin/io/spine/tools/gradle/JavaConfigurationName.kt

```diff
@@ -0,0 +1,229 @@
+         */
+        @JvmField
+        public val compileClasspath: ConfigurationName = compileClasspath(main)
+
+        /**
+         * Dependencies available at runtime but not at compile-time.
+         *
+         * Suitable for SPI implementations loaded via [java.util.ServiceLoader] or other
+         * classpath scanning utilities.
+         */
+        @Suppress("unused")
+        @JvmField
+        public val runtimeOnly: ConfigurationName = runtimeOnly(main)
+
+        /**
+         * All the dependencies included for the Java module runtime.
+         *
+         * Users cannot add dependencies directly to this configuration.
+         * However, this configuration may be resolved.
+         */
+        @JvmField
+        public val runtimeClasspath: ConfigurationName = runtimeClasspath(main)
+
+        /**
+         * The annotation processors used during the compilation of the tests of this module.
+         *
+         * These dependencies are not accessible to the user at compile-time or at runtime directly.
+         */
+        @JvmField
+        public val testAnnotationProcessor: ConfigurationName = annotationProcessor(test)
+
+        /**
+         * Dependencies on which the Java module tests rely for implementation.
+         *
+         * The dependencies are available at compile-time of the test code and at the test runtime.
+         */
+        @JvmField
+        public val  testImplementation: ConfigurationName = implementation(test)
+
+        /**
+         * All the dependencies included for the Java module tests compilation.
+         *
+         * Users cannot add dependencies directly to this configuration.
+         * However, this configuration may be resolved.
+         */
+        @Suppress("unused")
+        @JvmField
+        public val testCompileClasspath: ConfigurationName = compileClasspath(test)
+
+        /**
+         * Dependencies available at test runtime but not at compile-time.
+         *
+         * For example, JUnit runners may be depended on with this configuration.
+         */
+        @Suppress("unused")
+        @JvmField
+        public val testRuntimeOnly: ConfigurationName = runtimeOnly(test)
+
+        /**
+         * All the dependencies included for the Java module test runtime.
+         *
+         * Users cannot add dependencies directly to this configuration.
+         * However, this configuration may be resolved.
+         */
+        @JvmField
+        public val  testRuntimeClasspath: ConfigurationName = runtimeClasspath(test)
+
+        /**
+         * Configuration that allows to compile `.proto` files form the dependencies.
+         */
+        @JvmField
+        public val protobuf: ConfigurationName = name("protobuf")
+
+        /**
+         * A Spine-specific configuration used to download and resolve artifacts.
+         */
+        @JvmField
+        @io.spine.annotation.Internal
+        public val fetch: ConfigurationName = name("fetch")
```

Comment:
We may not need it anymore. It's fine for now, but as far as I can remember, we no longer need it in 2.x.

I would consider marking it `@Deprecated`.